databases/week1/hyf_lesson1.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(title) FROM task;
+SELECT*FROM task WHERE due_date IS NULL;
+SELECT task.id, task.title, status.name  FROM task, status WHERE status.name LIKE "%Done%";
+SELECT task.id, task.title, status.name FROM task , status WHERE status.name NOT LIKE "%Done%";
+SELECT task.id, task.title, task.created FROM task ORDER BY created DESC;
+SELECT task.id, task.title, task.created FROM task ORDER BY created DESC LIMIT 1;
+SELECT task.title,task.due_date,task.description from task WHERE task.title LIKE "%database%" OR task.description LIKE "%database%";
```

Comment:
Line 7. Great job! 
But consider the description is NOT required in the final out, just title and due_date. 
In this case, less is more. 👍